client/src/components/About.js

```diff
@@ -0,0 +1,179 @@
+                    }
+                    title="Elly S. Han"
+                  />
+                  <CardContent>
+                    Software engineer with a psychology background. With 8 years
+                    of experience at Microsoft, startup, and mHealth.
+                  </CardContent>
+                  <CardActions className={classes.actions}>
+                    <Button
+                      size="small"
+                      component={Link}
+                      to="https://www.linkedin.com/in/ellysalley/"
+                      color="primary"
+                    >
+                      LinkedIn
+                    </Button>
+                    <Button
+                      size="small"
+                      component={Link}
+                      to="https://github.com/ellysalley"
+                      color="primary"
+                    >
+                      GitHub
+                    </Button>
+                  </CardActions>
+                </Card>
+              </Grid>
+              <Grid key={1} item>
+                <Card className={classes.card}>
+                  <CardHeader
+                    avatar={
+                      <Avatar
+                        alt="Julian"
+                        src={Julian}
+                        className={classes.avartar}
+                      />
+                    }
+                    title="Julian J. Kohlman"
+                  />
+                  <CardContent>
+                    A persistent learner who enjoys solving problems from the
+                    backend, to the frontend.
+                  </CardContent>
+                  <CardActions className={classes.actions}>
+                    <Button
+                      size="small"
+                      component={Link}
+                      to="https://www.linkedin.com/in/julian-j-kohlman-563a758b/"
+                      color="primary"
+                    >
+                      LinkedIn
+                    </Button>
+                    <Button
+                      size="small"
+                      component={Link}
+                      to="https://github.com/juliankohlman"
+                      color="primary"
+                    >
+                      GitHub
+                    </Button>
+                  </CardActions>
+                </Card>
+              </Grid>
+              <Grid key={2} item>
+                <Card className={classes.card}>
+                  <CardHeader
+                    avatar={
+                      <Avatar alt="Sam" src={Sam} className={classes.avartar} />
+                    }
+                    title="Samuel Cha"
+                  />
+                  <CardContent>
+                    A software engineer that dreams of making “life” apps with
+                    rich websites and clean backends.
+                  </CardContent>
+                  <CardActions className={classes.actions}>
+                    <Button
```

Comment:
```html
<Button
  size="small"
  // component={Link}
  href="https://www.linkedin.com/in/chasoonjin/"
  target="_blank"
  color="primary"
>
```

I'm not sure if you had another way in mind, but this will open up the links in a new tab (the current copy doesn't seem to open it in a new tab right now)